compio/tests/io.rs

```diff
@@ -159,3 +165,179 @@ async fn writev_at() {
     assert_eq!(len, 3);
     assert_eq!(dst, [0, 0, 1, 1, 4]);
 }
+
+struct RepeatOne(u8);
```

Comment:
Consider adding this to `compio-io`?

Reply:
I don't think this struct is useful in most cases...